client/src/websocket/actions.ts

```diff
@@ -1,7 +1,7 @@
 import { HOSTNAME } from '../constants';
 import * as ActionTypes from './actionTypes';
 import { getWebSocket } from './selectors';
-import { Msg, ErrorMsg, ImageMsg, MsgType, VerificationFailMsg, CreateUserMsg } from '../message';
+import { ERROR, IMAGE, Msg, ErrorMsg, ImageMsg, TESTING, DRAWRESPONSE, VERIFICATIONFAIL, CREATEUSER } from '../message';
```

Comment:
I think that you shouldn't need to import all of these individually. The image team put all message types into an enum in a previous PR